Assets/LeapMotionModules/UI/Interaction/InteractionBehaviourBase.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Leap;
+using Leap.Unity.Attributes;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.UI.Interaction {
+
+  public abstract class InteractionBehaviourBase : MonoBehaviour {
+
+    public InteractionManager interactionManager;
+
+    protected Rigidbody _body;
+    public Rigidbody Rigidbody { get { return _body; } }
+
+    [SerializeField]
+    [HideInInspector]
+    #pragma warning disable 0414
+    // Variable used only for inspector rendering.
+    private bool _interactionManagerIsNull = true;
+    #pragma warning restore 0414
+
+    [Header("Interaction Settings")]
+    [DisableIf("_interactionManagerIsNull", isEqualTo: true)]
```

Comment:
You still want to be able to edit these fields even if a manager does not exist.  You might want to edit an interaction behaviour that is on a prefab, or before you create the manager.